index.html

```diff
@@ -0,0 +1,416 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta name="keywords" content="movies, theater, film, theatre">
+	<link rel="stylesheet" href="style.css">
+	<link rel="icon" type="image/png" href="http://example.com/myicon.png">
+	<title>Theater App</title>
+</head>
+
+
+<body>
+	<!--Make a header called Theater App name and a nav bar with different options user can click-->
+	<div class="main-wrapper">
+		<header>
+			<h1>Theater App</h1>
+			<nav class="navbar">
+				<ul>
+
+				</ul>
+			</nav>
+		</header>
+		<div class="main-content">
+			<main>
+				<!-- Image slider using js -->
+				<div class="img-slider"></div>
+				<!-- Browse currently playing button under the image slider -->
+				<section class="section section-now-playing">
+					<h2 class="heading-section">Now Playing</h2>
+
+					<div class="movie-list-wrapper">
+						<div class="movie-list">
+
+							<div class="movie">
+								<img
+									src="https://m.media-amazon.com/images/M/MV5BY2ZlNWIxODMtN2YwZi00ZjNmLWIyN2UtZTFkYmZkNDQyNTAyXkEyXkFqcGdeQXVyODkzNTgxMDg@._V1_.jpg"
+									alt="Movie cover image" class="movie__poster">
+								<div class="movie__details">
+									<h4 class="movie__title">Mortal Kombat</h4>
+									<span class="movie__rating">R</span>
```

Comment:
Everything honestly looks good from what I can see, my only small nitpick would be to scoot the "R" rating over a bit so it doesn't look so far away from everything else.